x/concentrated-liquidity/position.go

```diff
@@ -56,12 +55,12 @@ func (k Keeper) initOrUpdatePosition(
 		return types.NegativeLiquidityError{Liquidity: liquidity}
 	}
 
-	err = k.initOrUpdatePositionUptime(ctx, poolId, liquidity, owner, lowerTick, upperTick, liquidityDelta, joinTime, freezeDuration, positionId)
+	err = k.initOrUpdatePositionUptime(ctx, poolId, liquidity, owner, lowerTick, upperTick, liquidityDelta, joinTime, positionId)
```

Comment:
Copying from DMs:

From my understanding as it is in your PR - when update happens, the following occurs: 
- we update the position accumulator
- under the hood we claim rewards
    * however, I don't see logic for handling forfeited rewards there. Is the assumption that we are going to disable adding to a position in a separate PR?

If not, what is the right summary of the interaction during position update and incentive claiming/forfeit?

Reply:
Discussed this further in DMs, TLDR:

Every createPosition call will be unique, since joinTime will differ and a uniqueID will be assigned to it.